plugins/jira/help.md

```diff
@@ -1104,7 +1108,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
-* 6.3.1 - Fix Issue Where Create Issue failed when multiple versions of the input Issue Type exists in Jira | Fix failed connection test response for PAT based connection
+* 6.3.1 - Fix Issue Where Create Issue failed when multiple versions of the input Issue Type exists in Jira | Fix failed connection test response for PAT based connection | Add Include Fields input to triggers New Issue and Monitor Issues to include Issue fields in output | Removed empty Fields output from returned Issues when not requested or available
```

Comment:
Suggested re-word: Include Fields input added to New Issue and Monitor triggers, to specifies whether to return issue fields in the output.